.github/workflows/unit-tests.yml

```diff
@@ -0,0 +1,17 @@
+name: 'Run Unit Tests'
+
+on: 'push'
+
+jobs:
+  test:
+    runs-on: 'ubuntu-latest'
```

Comment:
```suggestion
    runs-on: ubuntu-22.04
```

Reply:
We use this specific LTS version of Ubuntu for the `ambi-rs` repo, so I think it's helpful to use the same version across the project.